build-system/tasks/dist.js

```diff
@@ -231,174 +224,135 @@ async function prebuild() {
   await preBuildWebPushPublisherFiles();
 }
 
-/**
- * Copies the css from the build folder to the dist folder
- * @return {!Promise}
- */
-function copyCss() {
-  const startTime = Date.now();
-
-  cssEntryPoints.forEach(({outCss}) => {
-    fs.copySync(`build/css/${outCss}`, `dist/${outCss}`);
-  });
-
-  return toPromise(
-    gulp
-      .src('build/css/amp-*.css', {base: 'build/css/'})
-      .pipe(gulp.dest('dist/v0'))
-  ).then(() => {
-    endBuildStep('Copied', 'build/css/*.css to dist/v0/*.css', startTime);
-  });
-}
-
 /**
  * Copies parsers from the build folder to the dist folder
- * @return {!Promise}
```

Comment:
same nit: async funcs return promises

Reply:
Same answer as above. We've gotten rid of `@return` annotations when we've gotten rid of explicit `return` statements in several other places in the codebase.